libs/core/src/main/java/org/opensearch/core/transport/TransportMessage.java

```diff
@@ -45,14 +45,24 @@ public abstract class TransportMessage implements Writeable {
 
     private TransportAddress remoteAddress;
 
+    private boolean sampled;
+
     public void remoteAddress(TransportAddress remoteAddress) {
         this.remoteAddress = remoteAddress;
     }
 
+    public void markResponseAsSampled() {
```

Comment:
It's confusing, which sampling are we talking here.

Reply:
This is Inferred sampling attribute. Let me rename this for better readability